config-service-api/src/main/proto/org/hypertrace/config/service/v1/config_service.proto

```diff
@@ -52,6 +55,9 @@ message UpsertConfigResponse {
 
   // prev version of config value in the store
   optional google.protobuf.Value prev_config = 4;
+
+  // user details
+  UserDetails user_details = 5;
```

Comment:
Now assuming it's the the user making the change this should be better named to indicate this goes with the update timestamp. But it also makes me wonder - should we be storing the creator too to keep these symmetrical?

Reply:
created and updated timestamp is already present in config document

Reply:
Right. I'm trying to get at the fact that the pattern is the same here. If we're updating this "user" on each write, it represents the user who last updated it only. If we're only persisting it on create but not update, it represents the user who created it only. 

If we only need one or the other that's fine, but since they're two different concepts (Creator and last updater) we should use the specific terminology vs a more ambiguous just "user"